portfolio/src/main/webapp/script.js

```diff
@@ -14,7 +14,7 @@
 
 // Fetches data from servelet when the document is loaded
 window.onload = async () => {
-  const response = await fetch('/data');
+  const response = await fetch('/comment');
   const jsonArray = await response.json();
```

Comment:
try/catch with error handling?

Reply:
It's in another branch! I just did a rebase